documentation/milestones/individual_milestones/communication_platforms.md

```diff
@@ -1,8 +1,19 @@
 # Communication platforms
 
+| Field           | Value          |
```

Comment:
should we add these fields to the template? I like them!

Reply:
Great suggestion! Added this in #26